schemas/participant.json

```diff
@@ -0,0 +1,137 @@
+{
+    "name": "participant",
+    "description": "The collection of data elements related to a specific donor in a PCGL program.",
+    "meta": {
+        "parent": "study"
+        },
+    "fields": [
+        {
+            "name": "study_id",
+            "valueType": "string",
+            "description": "Unique identifier of the study.",
+            "meta": {
+                "primaryId": true,
+                "examples": "TEST-CA",
+                "foreignKey": "study.study_id",
+                "notes": "Source:MOH/ARGO",
+                "displayName": "Study ID"
+            },
+            "restrictions": {
+                "required": true
+            }
+        },
+        {
+            "name": "submitter_participant_id",
+            "valueType": "string",
+            "description": "Unique identifier for the participant, assigned by the data provider.",
+            "meta": {
+                "primaryId": true,
+                "examples": "90234,BLD_donor_89,AML-90",
+                "displayName": "Submitter Participant ID",
+                "notes": "Source: MOH/ARGO;CQDG: submitter_participant_id;mCODE STU3: Patient.Identifer;Phenopacket: Individual.id; Beacon V2: Individual.id"
+            },
+            "restrictions": {
+                "required": true,
+                "regex": "#/regex/submitter_id"
+            }
+        },
+        {
+            "name": "sex_at_birth",
+            "description": "Refers to sex assigned at birth. Sex at birth is typically assigned based on a person's reproductive system and other physical characteristics. The provided values are based on the categoried defined by Statistics Canada",
+            "valueType": "string",
+            "meta": {
+                "core": true,
```

Comment:
Do we have a definition and meaning for what "core" field means? This feels like a carry over from ARGO that doesn't have a PCGL equivalent.